examples/ads.amp.html

```diff
@@ -457,6 +458,12 @@ <h2>AppNexus with JSON based configuration multi ad</h2>
       json='{"pageOpts":{"member": 958}, "adUnits": [{"disablePsa": true, "tagId": 6063968,"sizes": [300,250],"targetId": "apn_ad_1"}, {"tagId": 6063968,"sizes": [728,90],"targetId":"apn_ad_2"}]}'>
   </amp-ad>
 
+  <h2>AppVador</h2>
```

Comment:
I see "[APVAD] VAST Response is empty." when testing with this example, and I see a blank rect with text "filler tag". Wondering would it be more appropriate to use AMP provided `noContent` API in this case? 
Also we generally suggest adding rendering ad example (a fake one would be great) for testing purpose. Thank you.

Reply:
I replaced a fake one. Please check again.
Thank you.